v5 Electron Support (5-8)

glenn2223 opened the pull request:
Adding support for electron v5-8. PR'ing to v5 due to `node-gyp` bump

saper commented:
Thank you @glenn2223 for your patience! I'll check if my build machinery can build node-sass for electron as well.

glenn2223 commented:
Ref: Commit [Fix: Extensions don't honour argument `arch`](https://github.com/sass/node-sass/pull/2872/commits/7552f4e72acc8cc4c6aef43a4d17f9907b399143)

Test 1: [Set5: Line 4](https://github.com/glenn2223/node-sass/runs/526923273?check_suite_focus=true#step:5:4) setting arch as `ia32`
Test 2: [Success](https://github.com/glenn2223/node-sass/actions/runs/61416312)

___

Thanks @saper

nschonni commented on .github/workflows/electron-linux.yml:
```diff
@@ -0,0 +1,44 @@
+    - name: Use Node.js
+      uses: actions/setup-node@v1
+      with:
+        os: ${{ matrix.os }}
```
matrix.os doesn't look right, since there is no matrix in the build

glenn2223 replied:
👍
Left over from a file copy. Will remove

glenn2223 commented:
> This seems to have a lot more random formatting churn and changes to the overall build setup than the last PR. What has changed that required this?

* Formatting:  
Sorry I was working on another project at the same time so I just automatically hit my format shortcut. I can revert this if you wish?
* Fix: ([Fix: Extensions don't honour argument `target-arch`](https://github.com/sass/node-sass/pull/2872/commits/7552f4e72acc8cc4c6aef43a4d17f9907b399143))  
Did you want this on another PR for this?
* Electron targeting:
I wasn't able to find a way to run electron commands (for each version) from the CI so I added a `--electron-version` argument. This added a function and expanded a couple of lines that I've added. This also has the added benefit of targeting an electron build without electron
